src/System.Windows.Forms/src/System/Windows/Forms/LinkClickedEventArgs.cs

```diff
@@ -12,11 +12,38 @@ public class LinkClickedEventArgs : EventArgs
         /// <summary>
         ///  Initializes a new instance of the <see cref='LinkClickedEventArgs'/> class.
         /// </summary>
-        public LinkClickedEventArgs(string? linkText)
+        /// <param name="linkText">The text of the link being clicked.</param>
+        public LinkClickedEventArgs(string? linkText) : this(linkText, 0, 0) { }
+
+        /// <summary>
+        ///  Initializes a new instance of the <see cref='LinkClickedEventArgs'/> class.
+        /// </summary>
+        /// <param name="linkText">The text of the link being clicked.</param>
+        /// <param name="linkStart">The start of the link span being clicked.</param>
+        /// <param name="linkLength">The length of the link span being clicked.</param>
+        public LinkClickedEventArgs(string? linkText, int linkStart, int linkLength)
```

Comment:
```suggestion
        /// <exception cref="ArgumentOutOfRangeException">
        ///   <paramref name='linkStart' /> is negative or <paramref name='linkLength' /> is negative,
        ///   or the selection length is greater than <see cref="int.MaxValue"/>.
        /// </exception>
        public LinkClickedEventArgs(string? linkText, int linkStart, int linkLength)
```

@gewarren do we have a specific guidance on how to format "or" sentences?

Reply:
Yes, the guidance is here: https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions. 

Write each condition as a complete sentence, followed by a period, and separate the conditions with "-or-".